test/parse.js

```diff
@@ -142,6 +142,18 @@ test('string and alias', function (t) {
 	t.equal(typeof y.str, 'string');
 	t.equal(y.s, '000123');
 	t.equal(typeof y.s, 'string');
+
+	var z = parse(['-s123'], {
+		alias: { str: ['s', 'S'] },
+		string: ['str'],
+	});
+
+	t.equal(z.str, '123');
+	t.equal(typeof z.str, 'string');
```

Comment:
equal is strict, so we don’t need to check the type separately.

we’d also get a better error message if we used deepEqual on the entire z object.

Reply:
agree